core/src/test/scala/akka/persistence/jdbc/query/EventsByPersistenceIdTest.scala

```diff
@@ -171,8 +171,8 @@ abstract class EventsByPersistenceIdTest(config: String) extends QueryTestSpec(c
 
   it should "deliver EventEnvelopes non-zero timestamps" in withActorSystem { implicit system =>
     val journalOps = new ScalaJdbcReadJournalOperations(system)
+    val testStartTime = System.currentTimeMillis()
     withTestActors(replyToMessages = true) { (actor1, actor2, actor3) =>
-      val testStartTime = System.currentTimeMillis()
```

Comment:
This was a race before this fix as well, but the forced ping-pong with each actor makes it fail consistently each time now since actors are _always_ started when we get here now.